forcing_tools/netcdf2vic.py

```diff
@@ -0,0 +1,372 @@
+#!/usr/bin/env python
```

Comment:
I'm finding this naming convention to be a bit confusing. I think a more informative name for this would be `netcdf2ascii.py`.